packages/experience/src/pages/VerificationCode/index.tsx

```diff
@@ -21,9 +21,11 @@ const VerificationCode = () => {
   const { flow } = useParams<Parameters>();
   const { signInMethods } = useSieMethods();
 
-  const { identifierInputValue, forgotPasswordIdentifierInputValue } =
+  const { getIdentifierInputValue, forgotPasswordIdentifierInputValue } =
```

Comment:
Can directly use the original identiferInputValue here?

Reply:
Yes, since the identifier type and available sign-in methods are guarded on this page.

Reply:
Updated